.travis.yml

```diff
@@ -54,3 +50,13 @@ jobs:
               script: ./travis_wait "./include/travis/run_in_centos7_docker.sh ./include/travis/copr-deploy.sh build"
               on:
                   all_branches: true
+stages:
+    - test
+    - name: deploy-libzfs-sys
+      if: branch =~ ^v.+libzfs-sys$
+    - name: deploy-libzfs
+      if: branch =~ ^v.+\dlibzfs$
+    - name: deploy-node-libzfs
+      if: branch =~ ^v.+node-libzfs$
```

Comment:
Why doesn't this branch check for a number lik deploy-libzfs does?

Reply:
`deploy-libzfs` does the same thing.

I'll update these to look more like `deploy-copr`.